packages/ui/components/pages/Fuse/FusePoolsPage/FusePoolList/index.tsx

```diff
@@ -10,11 +20,41 @@ const FusePoolList = () => {
   const enabledChains = useEnabledChains();
   const { isLoading, poolsPerChain, error } = useCrossFusePools([...enabledChains]);
   const [err, setErr] = useState<Err | undefined>(error as Err);
+  const mounted = useRef(false);
+  const [preSearchText, setPreSearchText] = useState<string | undefined>(undefined);
+  const [preFilter, setPreFilter] = useState<(string | SupportedChains)[] | undefined>(undefined);
+  const [preSorting, setPreSorting] = useState<SortingState | undefined>(undefined);
+  const { address } = useMultiMidas();
 
   useEffect(() => {
     setErr(error as Err);
```

Comment:
Why set `err` when `error` is changing? `error` could be used directly from the `useQuery` `useCrossFusePools` hook.

Reply:
This `err` is for each chain, so for example if `polygon` got error, then `bsc` should be shown, so we should show each chain individually

Reply:
fixed